src/cookies.test.ts

```diff
@@ -95,6 +102,30 @@ describe('Cookies', () => {
                 cookieOptions,
             );
         });
+
+        // TODO: update test when PECR purposes introduced
+        it('is in preview', () => {
```

Comment:
You think it's worth adding a test to check it saves the cookie against the correct domain with a subdomain ending in `.com`? eg: `profile.theguardian.com` should be against the `.theguardian.com`.

Reply:
Discussed this point and reached the conclusion that the other tests already test for this case and they are sufficient.